static/css/main.css

```diff
@@ -90,32 +45,100 @@ header {
   margin: 5px auto;
 }
 
-/*MEDIA QUERIES*/
 
-@media (max-width: 870px) {
-  header {
-    /*height: 100px;*/
-    padding: 2em .8em 8em;
-  }
 
-  #nav {
-    float: left;
-  }
+/*=======Members section==============*/
+
+.members{
+  background-color: rgb(51, 51, 51);
+}
+
+.members__heading{
+  color: #fff;
+  font-family: Verdana, sans-serif;
+  margin-top: 30px;
+}
+
+.member{
+  max-width:260px;
+  margin-top: 50px;
```

Comment:
There seems to be too much space between the boxes. I suggest reducing the `margin-top` and `margin-bottom` to 10px.